kubernetes/resource_kubernetes_service_test.go

```diff
@@ -1165,15 +1171,17 @@ func testAccKubernetesServiceConfig_nodePort(name string) string {
     session_affinity = "ClientIP"
 
     port {
-      name        = "first"
-      port        = 10222
-      target_port = 22
+      name         = "first"
+      port         = 10222
+      target_port  = 22
+      app_protocol = "ssh"
     }
 
     port {
-      name        = "second"
-      port        = 10333
-      target_port = 33
+      name         = "second"
+      port         = 10333
+      target_port  = 33
+      app_protocol = "terraform.io/kubernetes"
```

Comment:
Extended `TestAccKubernetesService_nodePort` to verify `app_protocol` instead of creating a new test.